[IMP] stock_release_channel: Make _compute_picking_count hookable

mt-software-de opened the pull request.

TDu committed:
shopfloor_reception_mobile: imp states inheritance

TDu committed:
Add shopfloor_reception_packaging_dimension_mobile

TDu committed:
shopfloor: add packaging detail action

TDu committed:
shopfloor_reception: add hook for reception package dimension

TDu committed:
Add shopfloor_reception_packaging_dimension

jbaudoux reviewed (changes requested):
Don't change counter key names.

And don't add you as co-author, only contributor.

Otherwise looks good

jbaudoux commented:
Your PR is not on OCA

TDu committed:
sh_reception_packaging_dimension: refactor to allow module to extend

TDu committed:
sh_reception_packaging_dimension_mobile: refactor for extensibility

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (11 of 11 strings)

Translation: wms-14.0/wms-14.0-stock_storage_type_putaway_abc
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_storage_type_putaway_abc/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: wms-14.0/wms-14.0-shopfloor_batch_automatic_creation
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-shopfloor_batch_automatic_creation/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (15 of 15 strings)

Translation: wms-14.0/wms-14.0-stock_dynamic_routing_reserve_rule
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_dynamic_routing_reserve_rule/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (19 of 19 strings)

Translation: wms-14.0/wms-14.0-stock_reception_screen_measuring_device
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_reception_screen_measuring_device/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (27 of 27 strings)

Translation: wms-14.0/wms-14.0-stock_move_source_relocate
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_move_source_relocate/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 95.7% (45 of 47 strings)

Translation: wms-14.0/wms-14.0-stock_dynamic_routing
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_dynamic_routing/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (79 of 79 strings)

Translation: wms-14.0/wms-14.0-stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_available_to_promise_release/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (99 of 99 strings)

Translation: wms-14.0/wms-14.0-stock_reception_screen
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_reception_screen/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (304 of 304 strings)

Translation: wms-14.0/wms-14.0-shopfloor
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-shopfloor/it/

OCA-git-bot committed:
[IMP] update dotfiles

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (29 of 29 strings)

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (72 of 72 strings)

Translation: wms-14.0/wms-14.0-stock_warehouse_flow
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_warehouse_flow/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (79 of 79 strings)

Translation: wms-14.0/wms-14.0-stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_available_to_promise_release/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (99 of 99 strings)

Translation: wms-14.0/wms-14.0-stock_reception_screen
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_reception_screen/it/

TDu committed:
[IMP] manually fix linting

OCA-git-bot committed:
Merge PR #772 into 14.0

Signed-off-by jbaudoux

Commit:
[UPD] Update stock_dynamic_routing.pot

Commit:
[UPD] Update stock_move_source_relocate.pot

Commit:
[UPD] Update stock_storage_type.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: wms-14.0/wms-14.0-stock_storage_type
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_storage_type/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: wms-14.0/wms-14.0-stock_dynamic_routing
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_dynamic_routing/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: wms-14.0/wms-14.0-stock_move_source_relocate
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_move_source_relocate/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (8 of 8 strings)

Translation: wms-14.0/wms-14.0-stock_warehouse_flow_product_packaging
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_warehouse_flow_product_packaging/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (99 of 99 strings)

Translation: wms-14.0/wms-14.0-stock_reception_screen
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_reception_screen/it/

mt-software-de committed:
[IMP] stock_release_channel: Make _compute_picking_count hookable

OCA-git-bot committed:
Merge PR #647 into 14.0

Signed-off-by simahawk

Commit:
[UPD] Update shopfloor.pot

Commit:
[UPD] Update shopfloor_reception_packaging_dimension.pot

Commit:
[UPD] Update shopfloor_reception_packaging_dimension_mobile.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: wms-14.0/wms-14.0-shopfloor
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-shopfloor/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)
